src/components/flyout/flyout.tsx

```diff
@@ -301,10 +286,19 @@ export const EuiFlyout = forwardRef(
         closeButtonProps?.className
       );
 
+      const closeButtonStyles = euiFlyoutCloseButtonStyles(euiTheme);
+
+      const closeButtonCSSStyles = [
```

Comment:
Minor camel casing nit
```suggestion
      const closeButtonCssStyles = [
```